src/Defra.PTS.Web.UI/Helpers/BreedHelper.cs

```diff
@@ -21,7 +21,7 @@ public async Task<List<BreedDto>> GetBreedList(PetSpecies petType)
         var response = await _mediator.Send(new GetBreedsQueryRequest(petType));
 
         var list = response.Breeds.ToList();
-
+    
```

Comment:
Very picky but can we remove un-needed whitespace please?

Reply:
Done! @rCorcoran